csm_web/scheduler/tests/models/test_section.py

```diff
@@ -0,0 +1,77 @@
+import pytest
+from django.urls import reverse
+from scheduler.factories import (
+    CoordinatorFactory,
+    CourseFactory,
+    MentorFactory,
+    SectionFactory,
+    StudentFactory,
+    UserFactory,
+)
+from scheduler.models import Section
+
+
+@pytest.fixture(name="setup")
+def setup_test():
+    """
+    Create a course, coordinator, mentor, and a student for testing.
+    """
+    # Setup course
+    course = CourseFactory.create()
+    # Setup sections
+    section_one = create_section(course)
+    section_two = create_section(course)
+    # Setup students
+    create_students(course, section_one, 3)
+    create_students(course, section_two, 3)
+    # Setup coordinator for course
+    coordinator_user = UserFactory.create()
+    # Create coordinator for course
+    CoordinatorFactory.create(user=coordinator_user, course=course)
+
+    return (
+        section_one,
+        coordinator_user,
+    )
+
+
+@pytest.mark.django_db
+def test_section_delete(client, setup):
```

Comment:
There should be additional tests for edge cases, i.e. a section that the user is not a coordinator for, if the user is not a coordinator at all, if the section has existing students, etc.